docs/release-notes/workspace-namespace-binding.rst

```diff
@@ -0,0 +1,15 @@
+:orphan:
+
+**Breaking Changes**
+
+-  **Cluster:** The ``kubernetes_namespace`` field in the resource pool configuration is no longer
+   supported. Users can now submit workloads to specific namespaces by binding workspaces to
+   namespaces using the CLI or API.
+
```

Comment:
**Breaking Changes**

-  **Cluster:** The ``kubernetes_namespace`` field in the resource pool configuration is no longer
   supported. Users must submit workloads to specific namespaces by binding workspaces to
   namespaces using the CLI or API.

Reply:
Haven't adopted this change since user can use this feature if they want to but it is not required.

Reply:
@salonig23 is this resolved?